### PR TITLE
fix: Fix typehints for the HTTPClient 

### PR DESCRIPTION
**Pull request by sillyElvy**
## About

This pull request attempts to fix an issue where http typehints are not available 

## Checklist

- [ ] I've ran `pre-commit` to format and lint the change(s) made.
- [ ] I've checked to make sure the change(s) work on `3.8.6` and higher.
- [ ] This fixes/solves an [Issue](https://github.com/goverfl0w/discord-interactions/issues).
  - (If existent):
- [ ] I've made this pull request for/as: (check all that apply)
  - [ ] Documentation
  - [ ] Breaking change
  - [ ] New feature/enhancement
  - [x] Bugfix

### CODE REVIEW

File: interactions/api/http/client.pyi [hunk 1]
```diff
@@ -2,15 +2,41 @@ from typing import Optional, Tuple
...
 from ...api.cache import Cache
+from .channel import _ChannelRequest
+from .emoji import _EmojiRequest
+from .guild import _GuildRequest
+from .interaction import _InteractionRequest
+from .member import _MemberRequest
+from .message import _MessageRequest
+from .reaction import _ReactionRequest
+from .request import _Request
+from .route import Route
```
Comment (AstreaTSS) [inline interactions/api/http/client.pyi]: The `Route` import isn't necessary here.

**Review by AstreaTSS — CHANGES_REQUESTED** [review]
Seems good! Just one thing I noticed.

File: interactions/api/http/client.pyi (hunk 1 above)
Comment (sillyElvy) [reply interactions/api/http/client.pyi]: Tfw you copypasted

Comment (sillyElvy) [reply interactions/api/http/client.pyi]: Done

**Review by AstreaTSS — APPROVED** [review]

**Pull request merged**